### ISSUE TITLE
github: transfer govim repo to https://github.com/govim/govim?

### ISSUE THREAD

**Issue by myitcv** [govim/govim#473]
I've created https://github.com/govim as a placeholder. Feels like this repository would be better housed there.

Not least because we can then do small things like preventing people pushing commits to `master` (which is not possible with personal repos like this)

Should be able to transfer the repository I guess?

Points to consider:

* [x] Will the transfer automatically put in place repository redirects? - yes
* [x] Are there potentially any issues with the module changing path? - none
  * [x] For `cmd/govim`
  * [x] For the `github.com/myitcv/govim` package
* [x] update all references from `github.com/myitcv/govim` to `github.com/govim/govim`
* [x] update all wiki references from `github.com/myitcv/govim` to `github.com/govim/govim`
* [x] move repo
* [x] tag new release post module move
* [x] test `git clone` and `go get` of old and new paths
* [x] Fix Travis

cc @mvdan @leitzler @milesbxf for thoughts?

**Comment by leitzler** [comment]
Sounds good to me, we need a logo! :)

**Comment by mvdan** [comment]
Can you not accomplish the same with branch protection on your personal repo?

Up to you - I think this is personal preference. I generally only consider organizations when a project is either large or worked on actively by many people. It could also encourage others to take a more active role. So it seems fine to me :)

**Comment by myitcv** [comment]
> Can you not accomplish the same with branch protection on your personal repo?

Unfortunately not, just confirmed by GitHub support:

> That option is only available for Organization owned repositories, I'm afraid. It can't be used on repository belonging to an individual user account.

where "that option" is a reference to the "Restrict who can push to this branch" option on https://help.github.com/en/articles/enabling-branch-restrictions.

> I think this is personal preference

I'm not fussed either way. The transfer will be low cost now, higher cost in the future... doesn't seem to be any arguments against moving it... so let's go for it!

**Comment by ncw** [comment]
> I've created https://github.com/govim as a placeholder. Feels like this repository would be better housed there.

I think it is better for community engagement too.

> Points to consider:
> 
> * [ ]  Will the transfer automatically put in place repository redirects?

When I transferred the rclone repo everything was transferred, code, issues, prs, wiki, project.

Github put in redirects from the old repo to the new repo which worked perfectly.

I had to press the button to do the transfer twice - the first did nothing so I contacted github support and they said to do it again!

> * [ ]  Are there potentially any issues with the module changing path?
>   
>   * [ ]  For `cmd/govim`
>   * [ ]  For the `github.com/myitcv/govim` package
> * ...

Fixing up the code with s/oldrepo/newrepo/ was easy enough and module and non modules builds carried on working.

- https://github.com/rclone/rclone/commit/57d5de6fba12832469c27ee4eae6c2d4aa6a988f - fixing up the paths
- https://github.com/rclone/rclone/commit/f347514f62cc7b9f373520b72b05cfdb1e7de2ae - fixing CI badges

The CI was a bit more annoying and probably took most of the time getting it working properly again.  Travis required a lot of reauthenticating but I had to make a whole new account for Appveyor.  I think Azure pipelines was OK too.

I wrote a doc for contributors on how to fix their dev environments which you are welcome to steal

Note in particular the `git remote seturl` which is much better than using deleting the remote and readding it as it doesn't reset all the branch upstreams.

https://github.com/rclone/rclone/wiki/What-to-do-after-the-repository-move
